src/app/material-docs-app.html

```diff
@@ -1,2 +1,5 @@
+<div id="skip-link-wrapper">
```

Comment:
We like to uses classes over ids when possible because the same class can appear on more than 1 element and CSS rules targeted at classes are less specific than rules targeted at ids.